web2/htdocs/index.html

```diff
@@ -1445,6 +1461,25 @@
         break; /* #!/login */
         // }}}
 
+      case "#!/logout": // {{{
+        (function () {
+          api({
+            url: "/v2/auth/logout",
```

Comment:
You need a `type` here, of `GET`

Reply:
Are we changing the behavior of api in the future? GET is the documented default method of .ajax

Reply:
It's a good idea to be explicit when it aids understanding.